### ISSUE TITLE
Installer should ask for Current user vs All users

### ISSUE THREAD

**Issue by eirannejad** [eirannejad/pyRevit#205]
The user should be able to decide to install for the current user only in %appdata% folder or for all users on the machine.

**Comment by eirannejad** [comment]
Work in progress. Figured out how to make the custom dialog. Now off to figuring out how to use the collected data.

**Comment by eirannejad** [comment]
Resolved. 

**For GUI install:** The installer has the Current User / All Users option now and installation works

**For Silent install:** I could not figure out a way to pass an option to the installer for silent installs. However since the `install_addin.bat` now has a `--alluser` options, the admin can:

```
# Install using silent installer
pyRevitSetup-v4.exe /VERYSILENT /DIR="C:\pyRevit"

# then change directory to the release directory inside pyrevit repo
cd \D "C:\pyRevit\pyRevit\release"

# remove the current addin files
uninstall_addin.bat

# re-create the addin files using the --allusers option
install_addin.bat --allusers
```

**Comment by eirannejad** [comment]
Will be published in 4.4 soon
